environment/shell-configuration.rst

```diff
@@ -0,0 +1,13 @@
+######################
+Configuring your shell
+######################
+
+The `JupyterLab Terminal`_ runs the ``bash`` shell.
+You can configure your ``bash`` environment (including environment variables, aliases, and functions) by editing the :file:`~/.bashrc` or :file:`~/.bash_profile` files.
+
+.. note::
+
+   Environment variables that you set in :file:`~/.bashrc` or :file:`~/.bashrc_profile` are not available from notebooks.
```

Comment:
Good to know! I wonder if this can be repeated or linked somehow to the section on notebooks/.user_setups.